projects/1.19/assets/heat-and-climate/dcs_climate/lang/zh_cn.json

```
@@ -0,0 +1,824 @@
+  "block.dcs_climate.build.wall_schist_nether":"下界片岩墙",
+  "block.dcs_climate.build.wall_granite":"黑花岗岩墙",
+
+  "block.dcs_climate.build.stairs_beech_common":"山毛榉楼梯",
+  "block.dcs_climate.build.stairs_beech_walnut":"核桃楼梯",
+  "block.dcs_climate.build.stairs_beech_sweet":"白橡楼梯",
+  "block.dcs_climate.build.stairs_cherry_wild":"樱桃楼梯",
+  "block.dcs_climate.build.stairs_cinnamon_camphor":"香樟楼梯",
+  "block.dcs_climate.build.stairs_citrus_pomelo":"柑橘楼梯",
+  "block.dcs_climate.build.stairs_morus_mulberry":"桑木楼梯",
+  "block.dcs_climate.build.stairs_olive_ash":"白蜡楼梯",
+  "block.dcs_climate.build.stairs_palm_coconut":"棕榈楼梯",
+  "block.dcs_climate.build.stairs_reed_sorghum":"高粱茅草楼梯",
+
+  "block.dcs_climate.build.slab_beech_common":"山毛榉台阶",
+  "block.dcs_climate.build.slab_beech_walnut":"核桃台阶",
+  "block.dcs_climate.build.slab_beech_sweet":"白橡台阶",
+  "block.dcs_climate.build.slab_cherry_wild":"樱桃台阶",
+  "block.dcs_climate.build.slab_cinnamon_camphor":"香樟台阶",
+  "block.dcs_climate.build.slab_citrus_pomelo":"柑橘台阶",
+  "block.dcs_climate.build.slab_morus_mulberry":"桑木台阶",
+  "block.dcs_climate.build.slab_olive_ash":"白蜡台阶",
+  "block.dcs_climate.build.slab_palm_coconut":"棕榈台阶",
+  "block.dcs_climate.build.slab_reed_sorghum":"高粱茅草台阶",
+
+  "block.dcs_climate.build.fence_beech_common":"山毛榉栅栏",
+  "block.dcs_climate.build.fence_beech_walnut":"核桃栅栏",
+  "block.dcs_climate.build.fence_beech_sweet":"白橡栅栏",
+  "block.dcs_climate.build.fence_cherry_wild":"樱桃栅栏",
+  "block.dcs_climate.build.fence_cinnamon_camphor":"香樟栅栏",
+  "block.dcs_climate.build.fence_citrus_pomelo":"柑橘栅栏",
+  "block.dcs_climate.build.fence_morus_mulberry":"桑木栅栏",
+  "block.dcs_climate.build.fence_olive_ash":"白蜡栅栏",
+  "block.dcs_climate.build.fence_palm_coconut":"棕榈栅栏",
+  "block.dcs_climate.build.fence_reed_sorghum":"高粱茅草栅栏",
+
+  "block.dcs_climate.build.flintbricks":"燧石砖",
+  "block.dcs_climate.build.flintbricks_black":"黑色燧石砖",
+  "block.dcs_climate.build.flintbricks_white":"白色燧石砖",
+  "block.dcs_climate.build.flintbricks_red":"红色燧石砖",
+
+  "block.dcs_climate.build.mosaic_black":"黑色马赛克砖",
+  "block.dcs_climate.build.mosaic_blue":"蓝色马赛克砖",
+  "block.dcs_climate.build.mosaic_yellow":"黄色马赛克砖",
+  "block.dcs_climate.build.mosaic_red":"红色马赛克砖",
+
+    "## entities":"",
+
+  "entity.dcs_climate.bread_round":"圆面包",
+  "entity.dcs_climate.bread_square":"吐司面包",
+  "entity.dcs_climate.bread_cream":"奶油夹心面包",
+  "entity.dcs_climate.bread_sausage":"香肠面包",
+  "entity.dcs_climate.sandwich":"三明治",
+  "entity.dcs_climate.stick_beef":"牛肉串",
+  "entity.dcs_climate.stick_meat":"肉串",
+  "entity.dcs_climate.salad":"沙拉",
+  "entity.dcs_climate.soup":"汤碗",
+  "entity.dcs_climate.plate_steak":"烤牛排",
+  "entity.dcs_climate.plate_meat":"烤肉排",
+  "entity.dcs_climate.plate_legs":"烤腿排",
+  "entity.dcs_climate.plate_big_steak":"烤蒜香牛排",
+  "entity.dcs_climate.plate_chicken":"烤全鸡",
+  "entity.dcs_climate.plate_fish":"烤鱼排",
+
+    "##　tips":"",
+
+  "dcs.climate.tip.checker":"可以检测右击坐标的气候",

```

```suggestion
  "dcs.climate.tip.checker":"右击以检测该点的气候",
```
随你了（
此外日文键名是`dcs.tip.checker`